src/gei/Factories/GhesVersionCheckerFactory.cs

```diff
@@ -0,0 +1,16 @@
+﻿using OctoshiftCLI.GithubEnterpriseImporter.Services;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter.Factories
+{
```

Comment:
1. nit: use file scoped namespace
2. nit: so far we've put other factories in the root folder. for consistency my suggestion would be to either put this factory in the root directory (with changing the namespace of course) too or move others into the `Factories` folder.

Reply:
there's gotta be a way to make dotnet format catch the file scoped namespace for me. I'll do that, but I was gonna do a separate PR to move the other factories around (doing it immediately after this merges).

Reply:
I use `Rider` and it lets me know about those kind improvements 😃